dashboard_getting_started.md

```diff
@@ -0,0 +1,117 @@
+
+New framework users will need to sign up through the [Solar Forecast Arbiter
+dashboard](https://dashboard.solarforecastarbiter.org). The signup form can be
+found by following the 'Log in' link on the front page of the dashboard. You
+will be prompted with an Auth0 login window. Click the sign up tab and enter
+your information. Users will receive an email from Solar Forecast Arbiter to
+verify their account. Account verification is required before you will be able
+to access data in the Solar Forecast Arbiter. 
+
+Auth0 is a secure authentication service that has numereous security
+credentials and undergoes routine audits.
+
+### Organizations
+{:.anchor}
+
+Organizations are groups of users whose permissions and data are managed by
+the same group of administrators. An organization retains ownership of all data
+submitted by its members. Only organization administrators will be authorized
+to grant users permissions to view or alter data owned by their organization.
+
+#### Creating an Organization
+{: .anchor}
+
+When ready to create an organization, users should contact the framework
+administrators at [admin@solarforecastarbiter.org](mailto:admin@solarforecastarbiter.org)
+to start the process. The framework administrators will provide the
+non-negotiable [Data Use Agreement](/assets/45864%20Approved_Final%20version%201.1.pdf),
+the user will review the Data Use Agreement and return the Data Use Agreement
+signed by an authorized representative of their organization. A [point of contact](#organization-point-of-contact) will be established to verify future updates to the
+organization. An organization administrator will be appointed by the point of
+contact to manage data access.
+
+#### Joining an Organization
+{: .anchor}
+
+Users can join an existing organization by emailing the framework
+administrators at [admin@solarforecastarbiter.org](mailto:admin@solarforecastarbiter.org).
+Framework administrators will verify with the organization's point of contact
+that the user is authorized to join the organization. To expedite this process,
+an organization's point of contact may initiate a request to add the user to
+their organization.
+
+### User Classifications
+{: .anchor}
+
+Users of the Solar Forecast Arbiter will fall into one of the following
+classifications.
+
+#### Organization Point of Contact
+{:.anchor}
+
+An organization's point of contact is a representative of the organization who
+is authorized to approve or request changes to the organization. There may be
+more than one point of contact for an organization. A point of contact will be
+capable of contacting the framework administrators to complete the following
+actions:
+
+- Adding and removing users from the organization.
+
+- Promoting users to [organization administrators](#organization-administrators).
+
+- Participating in trials.
+
+#### Organization Administrators
+{: .anchor}
+
+Organization Administrators are users with special permissions to act on users,
```

Comment:
need a link to role based access control docs. maybe multiple links in this section.